README.md

```diff
@@ -278,6 +278,43 @@ manual folds from the syntax generated ones. More information about this
 problem and on how to configure the addon can be found on its GitHub
 page.
 
+## FAQ
+
+### How do enable/disable indenting in modules?
+There are two config variables for this. `b:verilog_indent_modules` (which will
+be eventually deprecated) and the newer `g:verilog_disable_indent`. Because of
+the existence of `b:verilog_indent_modules`, indenting in modules is disabled by
+default. As well as this, because `b:verilog_indent_modules` is a buffer
+vairable, to ensure indenting in modules is always enabled you must add
+the following to your .vimrc:
+```VimL
+    augroup systemverilog_settings_1
+        au!
+        au Filetype verilog_systemverilog let b:verilog_indent_modules = 1
+    augroup END
+```
+The same also applies for `b:verilog_indent_preproc` (indentation of
+preprocessor statements).
+
+### Why is opening verilog/systemverilog files so slow?
```

Comment:
The `Konfekt/FastFold` and `LargeFile` plugins take care of this Vim limitation. I could swear I had this information in the README, but now I can't find it there, nor on any git branch!

Could you please try these two plugins (start with FastFold alone first) to see if they work better than the solution you propose below?